File: app/src/main/java/kr/saintdev/pdiary/views/activitys/SplashActivity.java
package kr.saintdev.pdiary.views.activitys;

import android.annotation.SuppressLint;
import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.os.Message;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

import kr.saintdev.pdiary.MainActivity;
import kr.saintdev.pdiary.R;

public class SplashActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        Handler handler = new Handler() {
            public void handleMessage(Message msg) {
                super.handleMessage(msg);
                startActivity(new Intent(getApplicationContext(), MainActivity.class));
                finish();
            }
        };
        handler.sendEmptyMessageDelayed(0, 3000);
    }
}

File: app/src/main/java/kr/saintdev/pdiary/views/activitys/SplashActivity.kt
package kr.saintdev.pdiary.views.activitys

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import android.support.v7.app.AppCompatActivity
import kr.saintdev.pdiary.MainActivity
import kr.saintdev.pdiary.R

class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        val handler = Handler {
            startActivity(Intent(applicationContext, MainActivity::class.java))
            finish()
            true
        }
        handler.sendEmptyMessageDelayed(0, 3000)
    }
}